libbeat/cmd/instance/beat.go

```diff
@@ -123,6 +123,12 @@ var debugf = logp.MakeDebug("beat")
 
 func init() {
 	initRand()
+
+	// By default, the APM tracer is active. We switch behaviour to not require users to have
+	// an APM Server running, making it opt-in
+	if os.Getenv("ELASTIC_APM_ACTIVE") == "" {
+		os.Setenv("ELASTIC_APM_ACTIVE", "false")
+	}
```

Comment:
Nit: now that there are a couple of APM-related initialization statements here in the `init()` function, maybe move them into their own function like `initRand()`?